index.html

```diff
@@ -25,95 +25,333 @@
     />
     <link rel="stylesheet" href="style.css" />
   </head>
-
   <body>
-    <header>
-      <div class="container-header">
-        <a href="index.html" class="logo-title">Marlyn Mayienga</a>
-        <nav class="nav-bar">
-          <ul>
-            <li><a href="#">Portfolio</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contact</a></li>
-            <li>
-              <a href="#"><i class="far fa-envelope"></i></a>
-            </li>
-          </ul>
-        </nav>
-        <div class="hamburger-btn">
-          <i class="fas fa-bars"></i>
-        </div>
-      </div>
-    </header>
-
-    <section class="content">
-      <h1 class="name">Marlyn Mayienga</h1>
-      <img class="first-img" src="./circle_header.svg" alt="circle 2 header" />
-      <h2 class="dev-tag">Software Developer</h2>
-      <p class="first-paragraph">
-        I can help you build a product, feature or website. Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hestiate to contact me.
-      </p>
-      <hr class="first-line" />
-      <!-- TECHNLOGIES -->
-      <section class="technologies">
-        <div class="technologies-item">
-          <div class="technologies-title">Languages</div>
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>HTML</li>
-            <li>CSS</li>
-          </ul>
-        </div>
-        <div>
-          <div class="technologies-item">
-            <div class="technologies-title">Frameworks</div>
-            <ul>
-              <li>BootStrap</li>
-              <li>Ruby on Rails</li>
-              <li>RPSec</li>
-              <li>CapyBase</li>
-              <li>Selenium</li>
-            </ul>
-          </div>
-        </div>
-        <div>
-          <div class="technologies-item">
-            <div class="technologies-title">Skills</div>
+    <main>
+      <header>
+        <div class="container-nav">
+          <a href="index.html" class="logo-title">Marlyn Mayienga</a>
+          <nav class="nav-bar">
             <ul>
-              <li>Codekit</li>
-              <li>GitHub</li>
-              <li>CodePen</li>
-              <li>GitLab</li>
-              <li>Terminal</li>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li>
+                <a href="#"><i class="far fa-envelope"></i></a>
+              </li>
```

Comment:
- [ ] Kindly remove the underline from the nav links to match the Figma design.
- [ ] Kindly change the nav links color to white to match the Figma design.

Figma design
![image](https://user-images.githubusercontent.com/35535130/166837396-c0777e4e-ac26-45eb-871c-b682a9a55419.png)


Your design
![image](https://user-images.githubusercontent.com/35535130/166837439-33c5b8f9-7c4d-4a9f-8311-173246530a39.png)